rasa/core/channels/facebook.py

```diff
@@ -51,6 +51,15 @@ def _is_user_message(message: Dict[Text, Any]) -> bool:
             and not message["message"].get("is_echo")
         )
 
+    @staticmethod
+    def _is_quick_reply_message(message: Dict[Text, Any]) -> bool:
+        """Check if the message is a quick reply message."""
+        return (
+            message.get("message")
```

Comment:
let's make this a proper boolean expression (type checker likes that more)
```suggestion
            message.get("message") is not None
```